src/ratesConfig.ts

```diff
@@ -0,0 +1,41 @@
+import Big from "big.js";
+import debug = require("debug");
+import Asset from "./asset";
+const log = debug("bobtimus:rates");
+
+export interface TradeAmounts {
+  buyAsset: Asset;
+  sellAsset: Asset;
+  buyNominalAmount: Big;
+  sellNominalAmount: Big;
+}
+
+export type RatesConfig = {
+  [buyAsset in Asset]: {
+    [sellAsset in Asset]?: {
+      maxSell: number;
```

Comment:
:wrench: This name is confusing. Isn't this just a `rate`?

Reply:
We'll rename everything in the next PR that is rebased on this.

Reply:
I did some rename but more rename will come with the following PR that is already WIP